environment-wasm-host.yml

```diff
@@ -1,9 +1,9 @@
 name: xeus-cpp-wasm-host
 channels:
-  - https://repo.mamba.pm/emscripten-forge
+  - https://repo.prefix.dev/emscripten-forge-dev
   - https://repo.mamba.pm/conda-forge
 dependencies:
-  - emscripten-abi==3.1.45
+  - emscripten-abi==3.1.73
```

Comment:
Do we really need to pin anything here? I guess `3.1.73` is the only version available anyway.

Reply:
Hmmm, yes this shouldn't be necessary I think

Reply:
I mean, we could even remove this line I guess

Reply:
Cool, let me try that too !

Reply:
Merging as approved.